Index.html

```diff
@@ -81,6 +113,124 @@ <h3 class="work-subheading">Multi-Post Stories</h3>
     </ul>
     <button type="button" onclick="" class="work-button">See Project</button>
 
+    <div class="desktop-project-grid"> 
+      <div class="desktop-project-1">
+        <img class="project-background" src="images/desktop-project-background.png" alt="Project Background Image"/>
+        <button type="button" onclick="" class="project-button">See Project</button>
+      </div>
+
+      <div class="desktop-project-2">
+        <img class="project-background" src="images/desktop-project-background-2.png" alt="Project Background Image"/>
+        <h2 class="desktop-project-title-2">Data Dashboard <br> Healthcare</h2>
+        <p class="desktop-project-description-2">
+          A daily selection of privately <br> 
+          personalized reads, no accounts or <br> 
+          sign-ups required has been <br> 
+          the industry's standard.
+        </p>
+        <ul class="project-list">
+          <li class="project-item">
+            html
+          </li>
+          <li class="project-item">
+            bootstrap
+          </li>
+          <li class="project-item">
+            Ruby
+          </li>
+        </ul>
+      </div>
```

Comment:
Kindly fix the design of all the works cards so that whenever a user hovers on one, they can see the button that says "Show Project". This is not only the design of the first work, it's actually the design of all the work cards whenever a user hovers on it.

![image](https://user-images.githubusercontent.com/73714615/186402856-59b72f23-8eee-420b-807a-656759fe1169.png)

As you can see in the image above from Figma Design that the work card says "Show Project" when the mouse cursor hovers on it.